kv/kvnemesis: TestKVNemesisMultiNode_Restart_Liveness failed

kv/kvnemesis.TestKVNemesisMultiNode_Restart_Liveness [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsIbmCloudLinuxS390x_UnitTestsS390x/20633708?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsIbmCloudLinuxS390x_UnitTestsS390x/20633708?buildTab=artifacts#/) on master @ [04fd5d79281459419a7c6572aaa7f0cff8ef3f31](https://github.com/cockroachdb/cockroach/commits/04fd5d79281459419a7c6572aaa7f0cff8ef3f31):


```
google.golang.org/grpc.recv(0xc01699f470, {0x3ff4cf59978, 0xc000b368d0}, 0xc0189225a0, {0x0, 0x0}, {0x86057180, 0xc00a328690}, 0x7fffffff, 0xc014f87950, ...)
	external/org_golang_google_grpc/rpc_util.go:796 +0x84
google.golang.org/grpc.(*csAttempt).recvMsg(0xc017fab380, {0x86057180, 0xc00a328690}, 0x0)
	external/org_golang_google_grpc/stream.go:1080 +0x2d0
google.golang.org/grpc.(*clientStream).RecvMsg.func1(0xc017fab380)
	external/org_golang_google_grpc/stream.go:917 +0x42
google.golang.org/grpc.(*clientStream).withRetry(0xc018922360, 0xc014f87ac8, 0xc014f87ab8)
	external/org_golang_google_grpc/stream.go:752 +0x524
google.golang.org/grpc.(*clientStream).RecvMsg(0xc018922360, {0x86057180, 0xc00a328690})
	external/org_golang_google_grpc/stream.go:916 +0xd4
github.com/cockroachdb/cockroach/pkg/kv/kvpb.(*internalMuxRangeFeedClient).Recv(0xc00a7c2630)
	bazel-out/s390x-fastbuild/bin/pkg/kv/kvpb/kvpb_go_proto_/github.com/cockroachdb/cockroach/pkg/kv/kvpb/api.pb.go:11438 +0x6a
github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord.(*rangefeedMuxer).receiveEventsFromNode(0xc0109b2410, {0x874aef98, 0xc01891c280}, {0x3ff45b48bd0, 0xc00a7c2630}, 0xc01891c4b0)
	pkg/kv/kvclient/kvcoord/dist_sender_mux_rangefeed.go:459 +0x42
github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord.(*rangefeedMuxer).startNodeMuxRangeFeed(0xc0109b2410, {0x874aef98, 0xc01891c280}, {0x87481320, 0xc017327808}, 0x1, 0xc0027c5040)
	pkg/kv/kvclient/kvcoord/dist_sender_mux_rangefeed.go:419 +0xad4
github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord.(*rangefeedMuxer).establishMuxConnection.func1({0x874aef98, 0xc0109b1310})
	pkg/kv/kvclient/kvcoord/dist_sender_mux_rangefeed.go:350 +0x64
github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord.(*rangefeedMuxer).establishMuxConnection.Group.GoCtx.func2()
	pkg/util/ctxgroup/ctxgroup.go:200 +0xac
golang.org/x/sync/errgroup.(*Group).Go.func1()
	external/org_golang_x_sync/errgroup/errgroup.go:93 +0x60
created by golang.org/x/sync/errgroup.(*Group).Go in goroutine 134857
	external/org_golang_x_sync/errgroup/errgroup.go:78 +0xd0

goroutine 137376 [select]:
github.com/cockroachdb/cockroach/pkg/kv/kvclient/rangefeed/rangefeedcache.(*Watcher[...]).Run(0x874e6540, {0x874aef98, 0xc01ca5ab90})
	pkg/kv/kvclient/rangefeed/rangefeedcache/watcher.go:344 +0x9de
github.com/cockroachdb/cockroach/pkg/kv/kvclient/rangefeed/rangefeedcache.Start[...].func1()
	pkg/kv/kvclient/rangefeed/rangefeedcache/watcher.go:202 +0x1cc
github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunAsyncTaskEx.func1({0x874af4a0, 0xc00ced7180})
	pkg/util/stop/stopper.go:440 +0xac
created by github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunAsyncTaskEx in goroutine 130455
	pkg/util/stop/stopper.go:438 +0x16c

goroutine 135978 [select]:
github.com/cockroachdb/cockroach/pkg/kv/kvserver.(*baseQueue).processLoop.func2({0x874aefd0, 0xc01c2ac750})
	pkg/kv/kvserver/queue.go:1135 +0xfc
github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunAsyncTaskEx.func1({0x874aefd0, 0xc01c2ac750})
	pkg/util/stop/stopper.go:440 +0xac
created by github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunAsyncTaskEx in goroutine 135728
	pkg/util/stop/stopper.go:438 +0x16c

goroutine 131806 [select]:
github.com/cockroachdb/cockroach/pkg/kv/kvserver.(*baseQueue).processLoop.func2({0x874aefd0, 0xc020943e60})
	pkg/kv/kvserver/queue.go:1135 +0xfc
github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunAsyncTaskEx.func1({0x874aefd0, 0xc020943e60})
	pkg/util/stop/stopper.go:440 +0xac
created by github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunAsyncTaskEx in goroutine 132705
	pkg/util/stop/stopper.go:438 +0x16c
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
/cc @cockroachdb/kv-triage
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestKVNemesisMultiNode_Restart_Liveness.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)

</sub>


Jira issue: CRDB-55654

Test timed out after 5 minutes:

```
=== RUN   TestKVNemesisMultiNode_Restart_Liveness
    test_log_scope.go:171: test logs captured to: /artifacts/tmp/_tmp/ede2e2d7d4b16e91b4cc438bc4b2ba9a/logTestKVNemesisMultiNode_Restart_Liveness1754958068
    test_log_scope.go:82: use -show-logs to present logs inline
    kvnemesis_test.go:648: seed: 902598969553599692
-- Test timed out at 2025-10-22 13:15:00 UTC --
panic: test timed out after 5m0s
	running tests:
		TestKVNemesisMultiNode_Restart_Liveness (21s)
```

--- a/pkg/kv/kvnemesis/BUILD.bazel
+++ b/pkg/kv/kvnemesis/BUILD.bazel
@@ -62,7 +62,7 @@ go_library(
 
 go_test(
     name = "kvnemesis_test",
-    size = "medium",
+    size = "large",
     srcs = [
         "applier_test.go",
         "engine_test.go",

